tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/Support/OrderSummaryDataFactory.cs

```diff
@@ -0,0 +1,234 @@
+                        OrderItemEntityBuilder.Create()
+                            .WithOrderId(orderId)
+                            .WithOdsCode("Ods2")
+                            .WithCatalogueItemName("Order Item 2 ")
+                            .WithCatalogueItemType(CatalogueItemType.AssociatedService)
+                            .Build())
+                    .Build()
+                    .InsertAsync(_setting.ConnectionString)
+                },
+                {"complete-with-0recipient-1associatedservice-funding-complete", async (orderId) => await OrderSummaryDataBuilder.Create(orderId)
+                    .WithAssociatedServicesEntity(
+                        OrderItemEntityBuilder.Create()
+                            .WithOrderId(orderId)
+                            .WithOdsCode("Ods2")
+                            .WithCatalogueItemName("Order Item 2 ")
+                            .WithCatalogueItemType(CatalogueItemType.AssociatedService)
+                            .Build())
+                    .Build()
+                    .InsertAsync(_setting.ConnectionString)
+                },
+                {"complete-with-1solution-1associatedservice-funding-complete", async ( orderId) => await OrderSummaryDataBuilder.Create(orderId)
+                    .WithCatalogueSolutionEntity(
+                        OrderItemEntityBuilder.Create()
+                            .WithOrderId(orderId)
+                            .WithOdsCode("Ods1")
+                            .WithCatalogueItemName("Order Item 1 ")
+                            .WithCatalogueItemType(CatalogueItemType.Solution)
+                            .Build())
+                    .WithAssociatedServicesEntity(
+                        OrderItemEntityBuilder.Create()
+                            .WithOrderId(orderId)
+                            .WithOdsCode("Ods2")
+                            .WithCatalogueItemName("Order Item 2 ")
+                            .WithCatalogueItemType(CatalogueItemType.AssociatedService)
+                            .Build())
+                    .Build()
+                    .InsertAsync(_setting.ConnectionString)
+                },
+                {"complete-with-1solution-0associatedservice-funding-complete", async (orderId) => await OrderSummaryDataBuilder.Create(orderId)
+                    .WithCatalogueSolutionEntity(
+                        OrderItemEntityBuilder.Create()
+                            .WithOrderId(orderId)
+                            .WithOdsCode("Ods1")
+                            .WithCatalogueItemName("Order Item 1 ")
+                            .WithCatalogueItemType(CatalogueItemType.Solution)
+                            .Build())
+                    .Build()
+                    .InsertAsync(_setting.ConnectionString)
+                },
+                {"funding-incomplete", async (orderId) => await OrderSummaryDataBuilder.Create(orderId)
+                    .WithOrderEntity(
```

Comment:
For this scenario do we not just need to set the funding source to null? Could you get away without creating a new order & using the default?